src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/Prison.kt

```diff
@@ -0,0 +1,53 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity
+
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import org.hibernate.Hibernate
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "prison")
+class Prison(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val prisonId: Long = 0,
+
+  val code: String,
+
+  val description: String,
+
+  val enabled: Boolean,
+
```

Comment:
Worth a comment on the enabled attrib?  Enabled == Courts/probation cannot self-serve, but we will accept bookings from the prison (via DPS and A&A). Maybe enabled is the wrong name?

Reply:
Yes to comment. As for naming not sure, perhaps keep as is until you or I have an epiphany ...

Reply:
Done

Reply:
Enabled == false, means courts/probation cannot self-serve but will accept prison bookings from DPS/AA
(for clarity)